modules/market/src/main/java/com/opengamma/strata/market/curve/DefaultCurveMetadata.java

```diff
@@ -137,26 +164,27 @@ public DefaultCurveMetadata withParameterMetadata(List<CurveParameterMetadata> p
   private static final long serialVersionUID = 1L;
 
   /**
-   * Returns a builder used to create an instance of the bean.
-   * @return the builder, not null
+   * Creates an instance.
+   * @param curveName  the value of the property, not null
+   * @param xValueType  the value of the property, not null
+   * @param yValueType  the value of the property, not null
+   * @param info  the value of the property, not null
+   * @param parameterMetadata  the value of the property
    */
-  public static DefaultCurveMetadata.Builder builder() {
-    return new DefaultCurveMetadata.Builder();
-  }
-
-  private DefaultCurveMetadata(
+  DefaultCurveMetadata(
       CurveName curveName,
       ValueType xValueType,
       ValueType yValueType,
-      DayCount dayCount,
+      Map<CurveInfoType<?>, Object> info,
       List<? extends CurveParameterMetadata> parameterMetadata) {
     JodaBeanUtils.notNull(curveName, "curveName");
     JodaBeanUtils.notNull(xValueType, "xValueType");
     JodaBeanUtils.notNull(yValueType, "yValueType");
+    JodaBeanUtils.notNull(info, "info");
     this.curveName = curveName;
     this.xValueType = xValueType;
     this.yValueType = yValueType;
-    this.dayCount = dayCount;
+    this.info = ImmutableMap.copyOf(info);
```

Comment:
Should probably validate that the types of keys and values are compatible. We're just assuming they are. If they're not the failure won't happen until someone tries to look up a value.